iothub/service/src/Feedback/Models/FeedbackBatch.cs

```diff
@@ -8,28 +8,31 @@ namespace Microsoft.Azure.Devices
 {
     /// <summary>
     /// Contains a batch of feedback records.
-    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
     /// </summary>
+    /// <remarks>
+    /// Feedback messages are delivery acknowledgments for messages sent to a device from IoT hub.
```

Comment:
Extra info in remarks, not in summary. Also, no need to repeat for every member when the message can exist at the enum/class level.